docs/unified-test-documentation/dasharo-compatibility/343-port-PS2.md

```diff
@@ -0,0 +1,64 @@
+# Dasharo compatibility: Port PS/2
```

Comment:
@Pokisiekk please take into account existing test suites and cases: https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio/#sio001001-ps2-mouse-in-os-ubuntu-2204.
Will the PS/2 functionality on Qubes OS be different from other Linux-based OS in any way?

Reply:
I changed the name of the module. In theory, probably very similar, but I can't answer this question, because I haven't tested it on any of the OSes yet because of lack of hardware.